lib/ansible/module_utils/acme.py

```diff
@@ -46,7 +47,7 @@
     HAS_CURRENT_CRYPTOGRAPHY = (LooseVersion(CRYPTOGRAPHY_VERSION) >= LooseVersion('1.5'))
     if HAS_CURRENT_CRYPTOGRAPHY:
         _cryptography_backend = cryptography.hazmat.backends.default_backend()
-except Exception as _:
+except Exception as dummy:
```

Comment:
Note for the next time you need to do some work in here: rather than use a dummy variable, it's more idiomatic to just not assign the exception to a variable:
``` suggestion
except Exception:
```

Reply:
Does pylint/pep8 accept this style?

Reply:
@abadger thanks! I'll try to remember that :)
@MarkusTeufelberger it seems to be OK, considering that a lot of modules do that.

Reply:
Yep.  In default configurations, pylint will complain that Exception is caught (exception too broad).  It does not complain that the exception is not assigned to a variable. 

(Our configuration of pylint allows catching Exception.  Mostly because it would be extremely time consuming for us to clean that up... Having to evaluate every usage of Exception to see if it is valid or should be replaced with something more targeted)